R/git.R

```diff
@@ -143,17 +143,16 @@ git_commits <- function(branch, root = NULL) {
   commits
 }
 
-git_latest_commit <- function(root = NULL) {
-  args <- c("log", "--pretty='%h'", "-n", "1")
+git_latest_commit <- function(branch = "master", root = NULL) {
```

Comment:
I think we need to change this to look up the default branch otherwise we're going to hit problems. Can you look to see how widespread this is, and we might be better off doing it as a different PR.